command/runner.go

```diff
@@ -61,6 +66,17 @@ func Run(args []string, version string) error {
 
 	shouldColorize, subcommandInfo := ResolveSubcommand(args, config)
 
+	if config.Paging==kcconfig.PagingAlways && isOutputTerminal() && subcommandInfo.SupportsPager() {
```

Comment:
Instead of renaming the import to `kcconfig`, I suggest renaming this local variable to `cfg`